.eslintignore

```diff
@@ -33,5 +33,12 @@
 **/dist
 
 # Vendored files
-/packages/codemirror-graphql/
+**/graphiql.min.js
+**/graphiql.css
+**/graphiql.js
+
 /packages/graphiql/test/vendor
+
+# Funky codemirror-graphql build pattern
+/packages/codemirror-graphql/**
```

Comment:
You have to be very careful with this pattern; if this line matches a directory then files in that directory _will not be re-included_ by the `!` rule. ESLint uses the same algorithm as gitignore; which states:

> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. It is not possible to re-include a file if a parent directory of that file is excluded. Git doesn’t list excluded directories for performance reasons, so any patterns on contained files have no effect, no matter where they are defined.

To fix you have to match JUST the files explicitly:

```suggestion
/packages/codemirror-graphql/**/*.ts
/packages/codemirror-graphql/**/*.tsx
/packages/codemirror-graphql/**/*.js
/packages/codemirror-graphql/**/*.jsx
```

Maybe we should note this more strongly here: https://github.com/graphql/graphiql/blob/2da818b1f4e7adb1f1e519e314a428e846dc257e/.eslintignore#L7

Reply:
ok, the above suggestion works except it needs to ignore everything but `src` though. the whole issue with `codemirror-graphql` is that it builds to the root for easy resolutions, and the new eslintignore file doesnt ignore the built files anymore. the subpackage `.gitignore` honors this and we need the `.eslinignore` to do the same

Reply:
Yes; that's why I left the `!.../src/**` line below :) It's only the actual `/**` line I changed to make more explicit to only match files and not folders. I believe this will achieve the outcome you desire.